corehq/apps/hqadmin/views/users.py

```diff
@@ -431,7 +442,7 @@ def form_valid(self, form):
         )
         send_HTML_email(
             "%sYour account has been %s" % (settings.EMAIL_SUBJECT_PREFIX, verb),
-            self.user.get_email() if self.user else self.username,
+            self.user.email if self.user else self.username,
```

Comment:
This is in a weird state so I don't fault you for changing it, but I'm curious what prompted you to change it?

Reply:
`get_email()` is apparently not an attribute of `self.user` in this instance and shows a 500 error when you try to disable/enable accounts. It was more relevant when this page had the power to disable/enable accounts but I've since removed that and kept this bug fix.

Reply:
Ah it's a django user of course. Thanks for fixing! My one concern is that it is theoretically possible for the email field to be blank still, and we aren't accounting for that. A) it seems safe to assume `self.user` would exist at this point in this method so I don't think we need to check for that. B) we do have a `couch_user` available, so we could instead have:
```suggestion
            couch_user.get_email(),
```
What do you think?

Reply:
Yea, using couch_user here makes more sense - thanks for pointing that out!